src/main/java/Jelly/task/Event.java

```diff
@@ -54,6 +58,7 @@ public String toString() {
         }
         return "[E]" + super.toString() + "(from: " + fromWhen + " to: " + toWhen + ")";
     }
+
     @Override
     public String writeToFile() {
```

Comment:
Could be better named. 
I do not expect"writeToFile" to be a method that returns the String to be written